### PR TITLE
Clarify logging around TCK excludes files

### PR DESCRIPTION
**Pull request by gdams**
Currenty the test harness looks for an initial excludes file and the also attempts to detect an additional one. If the additional one doesn't exist then a misleading warning is shown that states `Unable to find excludes list file`.

This PR adds a check for the initial excludes file but also clarifies the logging

### CODE REVIEW

**Review by karianna — APPROVED** [review]
lgtm

File: jck/jtrunner/JavaTestRunner.java [hunk 1]
```diff
@@ -237,29 +237,32 @@ public static boolean generateJTB() throws Exception {
...
 		} else {
-			jtxRelativePath = "excludes/jck" + jckVersionNo + ".jtx";
-			kflRelativePath = "excludes/jck" + jckVersionNo + ".kfl";
+			System.out.println("Unable to find initial excludes list file " + initialJtxFullPath);
+			initialJtxFullPath = "";
 		}
-		
+
+		// Look for an update to the initial excludes file
+		jtxRelativePath = "excludes/jdk" + jckVersionNo + ".jtx";
```
Comment (Mesbah-Alam) [inline jck/jtrunner/JavaTestRunner.java:250]: This should be "excludes/jck" not "excludes/jdk". Same change is needed for the kfl file too, in the next line. 

(The exclude files under `JCK11-unzipped/tree/master/excludes` are : jck11.jtx & jck11.kfl).

Comment (Mesbah-Alam) [reply jck/jtrunner/JavaTestRunner.java:250]: An even simpler alternative for line 250 could be : `jtxRelativePath = "excludes/" + jckVersion + ".jtx"; `

Comment (gdams) [reply jck/jtrunner/JavaTestRunner.java]: Thanks @Mesbah-Alam. Updated PTAL

**Review by Mesbah-Alam — APPROVED** [review]
LGTM

**Comment by Mesbah-Alam** [comment]
Thanks @gdams !

**Pull request merged**